.github/workflows/deploy-sphinx-docs-on-ghpages.yml

```diff
@@ -37,42 +37,12 @@ jobs:
   build:
     
     runs-on: ubuntu-latest
```

Comment:
This one I believe you can drop, you're only interested in where the action called is running, not the caller :)

Reply:
I have to drop it. Thank you